pkg/manager/manager_integration_test.go

```diff
@@ -138,6 +138,20 @@ func (suite *ManagerTestSuite) TestLocalPods() {
 	assert.NoError(suite.T(), err)
 	// Pod cache should be empty because the node name is set to a nonexistent node.
 	assert.Empty(suite.T(), pods.Items)
+	assert.NoError(suite.T(), os.Setenv("NODE_NAME", nodeName))
+	node.SetKubernetesNodeName()
+}
+
+func (suite *ManagerTestSuite) TestGetNode() {
+	k8sNode, err := suite.manager.GetNode()
+	assert.NoError(suite.T(), err)
+	assert.Equal(suite.T(), nodeName, k8sNode.Name)
```

Comment:
nit: I think you might want to **require** no error or your second assert will panic on nil pointer